src/components/note/note.test.js

```diff
@@ -0,0 +1,178 @@
+import React from "react";
+import { EditorState, ContentState, convertFromHTML } from "draft-js";
+import Note from "./note.component";
+import Box from "../box";
+import LinkPreview from "../link-preview";
+import { ActionPopover, ActionPopoverItem } from "../action-popover";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import { tooltipPreview } from "../../../cypress/locators";
+import { linkPreview } from "../../../cypress/locators/link-preview/index";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+
+import {
+  noteComponent,
+  noteHeader,
+  noteFooterCreatedBy,
+  noteFooterChangeTime,
+  noteStatus,
+  noteContent,
+} from "../../../cypress/locators/note";
+import { actionPopoverWrapper } from "../../../cypress/locators/action-popover";
+
+const NoteComponent = ({ text, ...props }) => {
+  const initialValue = text
+    ? EditorState.createWithContent(ContentState.createFromText(text))
+    : EditorState.createEmpty();
+
+  return (
+    <Note
+      title="Here is a Title"
+      name="Lauren Smith"
+      noteContent={initialValue}
+      createdDate="23 May 2020, 12:08 PM"
+      {...props}
+    />
+  );
+};
+
+const NoteComponentWithInitialControls = () => {
```

Comment:
Should this const be titled 'NoteComponentWithInlineControls'? It might be a bit confusing for anyone looking at the test at a later date with the current title.